tests/evaluation/test_runner.py

```
@@ -0,0 +1,185 @@
+        assert "Pipeline timeout" in run.cases[1].error
+
+    async def test_persists_results_json(self, mock_pipeline_invoker, mock_scorer, tmp_path):
+        from evaluation.runner import EvaluationRunner
+
+        cases = _make_test_cases(2)
+        runner = EvaluationRunner(
+            pipeline_invoker=mock_pipeline_invoker,
+            scorer=mock_scorer,
+            output_dir=tmp_path,
+        )
+        run = await runner.run(cases, plugin_type="guidance", label="baseline")
```

Flake8 (7.3.0)

[error] local variable 'run' is assigned to but never used

(F841)